ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -30,6 +30,23 @@ export interface TimeSeriesChartOptions {
  */
 export function createInitialTimeSeriesChartOptions(): TimeSeriesChartOptions {
   return {
-    queries: [],
+    queries: [
+      {
+        kind: 'TimeSeriesQuery',
+        spec: {
+          plugin: {
+            kind: 'PrometheusTimeSeriesQuery',
+            spec: {
+              query: 'up',
```

Comment:
:+1:, we can remove this later

Reply:
yeah definitely, I should have left a TODO here too--let's sync up soon like you said, I imagine we're going to have a lot of merge conflicts soon